src/Razor/test/RazorLanguage.Test/IntegrationTests/ComponentCodeGenerationTestBase.cs

```diff
@@ -2202,7 +2202,7 @@ public void EventHandler_OnElement_WithEventArgsLambdaDelegate()
 
             // Act
             var generated = CompileToCSharp(@"
-<input @onclick=""@(x => { })"" />");
+<input @onclick=""x => { }"" />");
```

Comment:
@SteveSandersonMS - this just got a whole lot nicer but we sadly missed it in preview 6.